server/commands.lua

```diff
@@ -65,6 +65,35 @@ lib.addCommand('tp', {
     end
 end)
 
+lib.addCommand('bring', {
+    help = 'bring player to you',
+    params = {
+        { name = 'id', help = 'id to bring', optional = false },
+    },
+    restricted = "group.admin"
+}, function(source, args)
+    if not args['id'] or not tonumber(args['id']) then
+        TriggerClientEvent('QBCore:Notify', source, 'We need a valid numeric id to teleport to.', 'error')
+        return
+    end
+
+    local targetId = tonumber(args['id'])
+    local target = GetPlayerPed(targetId)
+
+    if target == 0 then
+        TriggerClientEvent('QBCore:Notify', source, Lang:t('error.not_online'), 'error')
+        return
+    elseif source == targetId then
+        TriggerClientEvent('QBCore:Notify', source, 'You cannot teleport to yourself.', 'error')
+        return
+    end
+
+    local playerPed = GetPlayerPed(source)
+    local coords = GetEntityCoords(playerPed)
+    TriggerClientEvent('QBCore:Command:TeleportToPlayer', targetId, coords)
```

Comment:
Would it be better to set the target coords directly here instead of calling this event? We can just set the coordinate from the server side.

Reply:
Yeah i agree I'll make that change